File: MRSource/Framework/DBTimeLine/Framework.DBTimeLine/DBCustomizationPersister.vb
Imports MRFramework

Public Class DBCustomizationPersister
    Inherits MRPersisting.MRPersister

    Public Overrides ReadOnly Property DataBaseTableName As String
        Get
            Return "Config.Customization"
        End Get
    End Property
    Public Overrides ReadOnly Property SQL As String
        Get
            Return "SELECT ID, CustomizationKey, Created, Active, Description FROM " & DataBaseTableName
        End Get
    End Property
End Class

File: MRSource/Customizations/Customizations.Core/Persisters/DBCustomizationPersister.cs
namespace Customizations.Core.Persisters
{
    class DBCustomizationPersister : Framework.Persisting.Persister
    {
        protected override string DataBaseTableName
        {
            get
            {
                return "Config.Customization";
            }
        }

        protected override string Sql
        {
            get
            {
                return "SELECT * FROM " + DataBaseTableName;
            }
        }
    }
}
